Composer/packages/lib/shared/src/dialogFactory.ts

```diff
@@ -1,40 +1,50 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import nanoid from 'nanoid/generate';
+import { JSONSchema7 } from 'json-schema';
+import merge from 'lodash/merge';
 
 import { DesignerData } from './types/sdk';
-import { appschema } from './appschema';
 import { copyAdaptiveAction } from './copyUtils';
 import { deleteAdaptiveAction, deleteAdaptiveActionList } from './deleteUtils';
 import { MicrosoftIDialog } from './types';
 import { SDKTypes } from './types';
 import { ExternalResourceHandlerAsync } from './copyUtils/ExternalApi';
+import { generateUniqueId } from './generateUniqueId';
 
 interface DesignerAttributes {
   name: string;
   description: string;
 }
 
 const initialInputDialog = {
-  allowInterruptions: 'false',
+  allowInterruptions: false,
   prompt: '',
   unrecognizedPrompt: '',
   invalidPrompt: '',
   defaultValueResponse: '',
 };
 
-const initialDialogShape = {
-  [SDKTypes.AdaptiveDialog]: seededActions => ({
+export function getNewDesigner(name: string, description: string) {
+  return {
+    $designer: {
+      name,
+      description,
+      id: generateUniqueId(6),
+    },
+  };
+}
+
+const initialDialogShape = () => ({
```

Comment:
This needs to be a function in order to get unique ids every time this is called.